issue_drafts/issue_0.md

```diff
@@ -1,3 +1,38 @@
 # Issue 0: Installing Snakemake
+In order to walk through this learning module, we will create a new python environment and install snakemake and other required packages.
 
+Please make sure you have anaconda installed on your computer before proceeding with the following steps.
 
+If you are working in Windows, you will need to open your Anaconda prompt to follow the steps below. If you are working in Linux or macOS, you can just open your command line.
+
+## Install using environment.yaml
+You can create an anaconda environment for this tutorial with all the required packages in one step by running the following command:
+`
+conda env create -f environment.yaml
+`
+
+Note: You may have issues installing with the environment.yaml file. This could be due to the fact that anaconda sometimes has trouble installing snakemake. If this is the case, follow the instructions below for a manual installation.
+
+## Create environment and install packages manually
+You can manually create your anaconda environment and install the required packages by following the steps below.
+
+1. Create a new conda environment: `conda create -n snakemake-tutorial python=3.8`
+
+2. Activate the environment: `conda activate snakemake-tutorial`
+
+
+3. Install mamba, a fast and robust replacement for the conda package manager that is better able to handle the installation of snakemake: `conda install -c conda-forge mamba`
+
+4. Install snakemake (please [read the docs](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html) for more details on installation). If you are working in a Mac or Linux OS, you can install the full version of snakemake: `mamba install -c conda-forge -c bioconda snakemake`. If you are working in Windows, some of the dependencies in the full version of snakemake will not work for you. You can install the minimal version of snakemake:`mamba install -c conda-forge bioconda::snakemake-minimal`
+
+5. Install any additional packages needed for this tutorial (listed in the environment.yaml file) using conda or mamba.
+
+
+
```

Comment:
It would be good to include a test to make sure that Snakemake has been installed, like
```
snakemake --help
```

Reply:
good point! I added a few changes as in-line suggestions if you want to take a look and commit them!